themes/jaeger-docs/layouts/shortcodes/dockerImages.html

```diff
@@ -1,4 +1,6 @@
 {{- $dockerImages := .Site.Data.download.docker }}
+{{- $versions     := sort (readDir "content/docs") "Name" "desc" }}
+{{- $latest       := (index ($versions) 0).Name }}
```

Comment:
no way to DRY this?

Reply:
I suspect that you can't really cut down on repetition without introducing a bunch of extra complexity. But I'll think on it.